exosphere-shared/src/docker-helper.ls

```diff
@@ -9,12 +9,18 @@ class DockerHelper
     child_process.exec-sync('docker ps -a --format {{.Names}}', 'utf8') |> (.includes container)
 
 
+  @ensure-container-is-running = (container-name, image) ->
+    return if child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}' , 'utf8') |> (.includes "#{container-name}/Up")
+    return child_process.exec-sync("docker start #{container-name}") if @container-exists container-name
+    @run-image container-name, image
```

Comment:
Does it make sense to wait here until the image is actually running, or should that happen somewhere else?